apps/www/_blog/2023-04-10-o11y-day.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: Supabase Logs are now self-hosted
+launchweek: 7
+tags:
+  - launch-week
+  - o11y
+date: '2023-04-10'
+to_depth: 3
+author: chasers,ziinc
+---
+
+Today, we’re releasing Supabase Logs for both self-hosted users and CLI development
+
+## Logflare Introduction
+
+Since [Logflare joined Supabase](https://supabase.com/blog/supabase-acquires-logflare) over a year ago it’s been quietly handling over 1 billion log events every day. These events come from various tools in the Supabase infrastructure - the API gateway, Postgres databases, Storage, Edge Functions, Auth, and Realtime.
+
+Logflare is a multi-node, highly available Elixir cluster, ingesting the log events and storing them into BigQuery for Supabase and Logflare’s customers. On average, the cluster has 6 nodes handling every spike our customers throw at it.
+
+To expose log data to customers, we leverage Logflare Endpoints. This provides an HTTP integration into Supabase Studio, powering the log query UIs and most timeseries charts. These charts live across the studio, such as the project home page and the new API reports.
+
+{/* <!--  add chart screenshot --> */}
+
+## Self-hosting Logflare
+
+Logflare was available under a BSL license prior to joining Supabase. We’ve since changed the license to [Apache 2.0](https://github.com/Logflare/logflare/blob/staging/LICENSE.md), aligning it with our open source philosophy.
+
+In the past few months we’ve made Logflare more developer-friendly for local development and self-hosting. While you’re building a project, you can view and query your logs from any Supabase service, just as you would in our cloud Platform.
+
+![Realtime local logs](/images/blog/lw7-o11y/realtime-logs-screenshot.png)
+
+> 📢 Check out the [new self-hosting docs](http://localhost:3001/docs/reference/self-hosting-analytics/introduction) to get Logflare up and running as your analytics server.
+>
+> It currently supports a BigQuery backend, and we are actively working on supporting more.
```

Comment:
```suggestion
> 📢 Check out the [new self-hosting docs](http://localhost:3001/docs/reference/self-hosting-analytics/introduction) to get Logflare up and running as your analytics server. It currently supports a BigQuery backend, and we are actively working on supporting more.
```

side note @MildTomato - do we have an "Callout" component that we could use here? A bit strange as a quote

Reply:
yea, i'll take a look

Reply:
<img width="1262" alt="Screenshot 2023-04-10 at 5 31 47 PM" src="https://user-images.githubusercontent.com/8291514/230875366-352601e7-eab3-4fc0-933d-af8a03f63b2e.png">

changed to above for now.
we should get the same Admonitions setup in both docs and www so you can just do 

```
:::note

Some **content** with _Markdown_ `syntax`. Check [this `api`](#).

:::
```

Reply:
fixed